keywords/information-overload.md

```diff
@@ -0,0 +1,65 @@
+# Information Overload
```

Comment:
To maintain the same format, and according to the instructions, this should be a heading 2 (i.e., `##`)